packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -487,7 +487,7 @@ storiesOf('Picker', module)
   .add(
     'isLoading more',
     () => (
-      <Picker label="Test" isLoading items={flatOptions}>
+      <Picker label="Test" isLoading isOpen items={flatOptions}>
```

Comment:
why is this needed

Reply:
Ah lol, forgot to remove, was testing something